modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -106,6 +108,80 @@ function BulkActions({
 		}
 	}
 
+	const renderNewBulkActions = ({
+		formId,
+		formName,
+		loadData,
+		namespace,
+		sidePanelId,
+	}) => {
+		return (
+			<ul className="bulk-actions navbar-nav">
+				{bulkActions.map((actionDefinition, i) => (
+					<li className="nav-item" key={actionDefinition.label}>
+						<button
```

Comment:
Can we use `ClayButton` here, or `ClayButtonWithIcon`?

Reply:
This part is duplicating the markup for the other "highlighted" actions so I wanted to leave them untouched for this ticket. https://github.com/liferay-frontend/liferay-portal/blob/a0f2e0c73eccca9c9dd011b767e0922787aa383f/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js#L268-L291